source/api_reference/index.html.md.erb

```diff
@@ -104,7 +102,7 @@ A successful response includes ``status`` and ``finished`` values:
   </tr>
   <tr>
     <td> error </td>
-    <td> Something went wrong with GOV.UK Pay or the underlying payment service provider. <a href="/support_contact_and_more_information/#contact-us">Contact us.</a> </td>
+    <td> Something went wrong with GOV.UK Pay or the underlying payment service provider. Payment fails safely with no money taken. <a href="/support_contact_and_more_information/#contact-us">Contact us.</a> </td>
```

Comment:
what do we show the user here? and error message as well?

Reply:
I don't know. Maybe we should put a link to the HTTP status code content below this table?

Reply:
@sfount @kbottla can you advise?

Reply:
Users will see below, if there is a technical error
<img width="504" alt="Screenshot 2020-11-18 at 17 15 43" src="https://user-images.githubusercontent.com/40598480/99563942-b8282e00-29c1-11eb-8042-e465486b089e.png">

We display this hint in selfservice for Error status

```
The was an error with the payment. This could be due to a configuration problem between the service and the payment provider. Contact the GOV.UK Pay team with the payment ID, quoting error code P0050.
```

Reply:
thank you!